openvdb_ax/openvdb_ax/compiler/Logger.h

```diff
@@ -197,7 +199,9 @@ class OPENVDB_AX_API Logger
 
 private:
 
-    friend class ::TestLogger;
+    FRIEND_TEST(TestLogger, testParseSetsTree);
+    FRIEND_TEST(TestLogger, testAddError);
+    FRIEND_TEST(TestLogger, testAddWarning);
```

Comment:
It might have been possible to avoid these changes to a header file in order to test private members, but this is the GTest- recommended approach. See https://google.github.io/googletest/advanced.html#testing-private-code